src/main/java/org/broadinstitute/hellbender/tools/exome/GetHetCoverage.java

```diff
@@ -98,16 +99,19 @@
     )
     protected int minimumBaseQuality = 20;
 
+    @Argument(
+            doc = "Validation stringency for all BAM files read by this program.  Setting stringency to SILENT " +
+            "can improve performance when processing a BAM file in which variable-length data (read, qualities, tags) " +
+            "do not otherwise need to be decoded.",
+            common=true)
```

Comment:
i just copied the VALIDATION_STRINGENCY argument parameters almost word-for-word from the analogous code in public.  according to the Argument documentation, common=true if "this an Option common to all command line programs.  If it is then it will only be displayed in usage info when H or STDHELP is used to display usage."

once LocusWalker/pileup is implemented, i think we'll be able to remove this duplicated code.